code/modules/antagonists/heretic/heretic_antag.dm

```diff
@@ -23,6 +23,7 @@
 	hijack_speed = 0.5
 	suicide_cry = "THE MANSUS SMILES UPON ME!!"
 	preview_outfit = /datum/outfit/heretic
+	var/list/all_sac_targets = list()
```

Comment:
I'd ask that you put this down at `var/list/mob/living/carbon/human/sac_targets`
Also auto-doc

Reply:
whats auto doc